src/app/admin/admin.component.html

```diff
@@ -0,0 +1,57 @@
+<div>
```

Comment:
Use 4 space indentation instead of 8 space indentation like we have been following for all html files till now. That's the standard.
We need to update it in coding guidelines as well.